File: python/gastoCombustivel.py
tempoViagem = int(input('Digite o tempo de viagem em horas: '))
velocidadeMedia = int(input('digite a velocidade media em km/h: '))

distancia = velocidadeMedia * tempoViagem
litros = float(distancia / 12)

print(f'a quantidade de litros para a viagem é {litros:,.3f}')

File: js/exercicios/gastoCombustivel.js
const input = require('../node_modules/readline-sync');

const tempoViagem = input.question('Digite o tempo de viagem em horas: ');
const velocidadeMedia = input.question('digite a velocidade media em km/h: ');

const distancia = velocidadeMedia * tempoViagem;
const litros = distancia / 12;

console.log(`${litros.toFixed(3)}`);

